rustler_sys/build.rs

```diff
@@ -27,12 +27,13 @@ fn try_gen_api(dst: &Path, pointer_size: usize) -> bool {
 }
 
 fn main() {
-    // get size of C pointer
-    let target_pointer_width = match env::var("CARGO_CFG_TARGET_POINTER_WIDTH") {
-        Ok(ref val) if val == "32" => 4,
-        Ok(ref val) if val == "64" => 8,
-        Ok(ref val) => panic!("Unsupported target pointer width: {}", val),
-        Err(err) => panic!(
+    // get size of C long
+    let long_width = match (env::var("CARGO_CFG_TARGET_POINTER_WIDTH"), env::var("CARGO_CFG_TARGET_OS")) {
```

Comment:
Maybe pull these into vars above to clean up the formatting?

Reply:
Sure.